hnn_core/tests/test_network.py

```diff
@@ -54,6 +54,25 @@ def test_network():
                                                 n_gaus_sources +
                                                 n_common_sources)
 
+    # Assert that netcons are created properly
+    # proximal
+    assert 'L2Pyr_L2Pyr_nmda' in network_builder.ncs
+    n_pyr = len(net.gid_dict['L2_pyramidal'])
+    n_connections = 3 * (n_pyr ** 2 - n_pyr)
```

Comment:
I'm probably missing something obvious, but why is there a factor of 3 here?

Reply:
it's because there are 3 synapses per cell which are connected for NMDA.

Reply:
Oh that makes so much sense!